site/includes/contributors.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="row">
+	<h2 class="wb-inv">
+		{{#is page.language "fr"}}
+		Collaborateurs
+		{{else}}
+		Contributors
+		{{/is}}
+	</h2>
+	<section class="provisional gc-contributors col-md-12">
+		<div class="col-md-1">
+			<h3 class="provisional from">
```

Comment:
"from" css is not needed, please scope the style by using the component pattern main identify identified by "gc-contributors" in combinaison of "provisional"

Reply:
Are you saying to add to the CSS provisional.gc-contributors h3 what is currently in the "from" class?